src/modules/home/containers/homeScreen.js

```diff
@@ -31,6 +31,7 @@ class HomeContainer extends Component {
         navigateToScan={this.navigateTo('BookScanner')}
         navigateToProfile={this.navigateTo('Profile')}
         navigateToShoppingBag={this.navigateTo('ShoppingBag')}
+        navigation={this.props.navigation}
```

Comment:
Not ideal. Nao sei porque nao tava funcionando pegar a referencia do navigation direto do outro container